emukit/core/optimization/random_search_acquisition_optimizer.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from typing import Tuple
+
+import numpy as np
+from GPyOpt.optimization.acquisition_optimizer import ContextManager
+
+from .. import ParameterSpace
+from ..acquisition import Acquisition
+from ..optimization.acquisition_optimizer import AcquisitionOptimizerBase
+
+_log = logging.getLogger(__name__)
+
+
+class RandomSearchAcquisitionOptimizer(AcquisitionOptimizerBase):
+    """ Optimizes the acquisition function by evaluating at random points."""
+    def __init__(self, space: ParameterSpace, num_samples: int) -> None:
+        """
+        :param space: The parameter space spanning the search problem.
+        :param num_samples: Number of random sampled points which are evaluated per optimization.
```

Comment:
I find "num_samples" name confusing in this context. Maybe "num_eval_points"? Very subjective though

Reply:
True these are poorly named. same for the local search.